### ISSUE TITLE
WER for GigaSpeech

### ISSUE THREAD

**Issue by Freddy-pp** [espnet/espnet#3360]
WER results on Espnet recipe page ( https://github.com/espnet/espnet/tree/master/egs2/gigaspeech/asr1#wer ) and on the corpus page ( https://github.com/SpeechColab/GigaSpeech#leaderboard ) are different.
WER dev - 9.2 
WER test - 15.6
on Espnet page, and
Dev/Test WER - Espnet - 10.90 / 10.80
on GigaSpeech page.

Words count from the resulting table on Espnet page also different when comparing to results for other toolkits.
Espnet - dev/test words - 51075/175116 https://github.com/espnet/espnet/tree/master/egs2/gigaspeech/asr1#wer
Kaldi - dev/test words - 127790/390721 - https://github.com/kaldi-asr/kaldi/blob/master/egs/gigaspeech/s5/RESULTS
Wenet - dev/test words - 127790/390656 - https://github.com/wenet-e2e/wenet/tree/main/examples/gigaspeech/s0

Do I understand correctly that the dev/test sets are different in these cases? And are there any results on original GigaSpeech Dev/Test sets for espnet model, and do they the same as on the GigaSpeech page (10.90 / 10.80)?

**Comment by sw005320** [comment]
Thanks a lot for your report.
We used the very old version, and I’m expecting that the test set was changed since then.
@chenguoguo, could you check this?
If the training set was also changed, I’ll re-run the entire experiments.

**Comment by chenguoguo** [comment]
We found a bug in the dev/test splitting and we fixed that before the release. The training set was untouched, so @sw005320 you don't have to re-train the models. In fact, Shinji, I emailed you on this before the paper deadline and we got the 10.90 / 10.80 numbers on that exact same model.

We have asked @chaisz19 to submit a PR to Espnet to:
1. Add the GigaSpeech normalization or use the GigaSpeech scoring (https://github.com/SpeechColab/GigaSpeech/blob/main/utils/gigaspeech_scoring.py)
2. Fix the numbers

I'll ping her on this.

**Comment by sw005320** [comment]
Thanks!
> In fact, Shinji, I emailed you on this before the paper deadline and we got the 10.90 / 10.80 numbers on that exact same model

Sorry that I forgot it….
Maybe, I can dig my directories, and update the result part at least.
I’m on the vacation, but I’ll make a follow up PR after @chaisz19 makes some actions.

**Comment by BuaaAlban** [comment]
Have you test the model on Librispeech and could you supply the wer if you have tested it?

**Comment by sw005320** [comment]
As far as I remember, we performed such a test.
@chenguoguo, is it correct? Do you have the result? I checked my email but I could not find it....
